projects/nano-editor.org/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://www.nano-editor.org/dist/v{{ version.major }}/nano-{{ version.raw }}.tar.gz
+
+versions:
+  - 7.2
+
+dependencies:
+  gnu.org/gettext: "*"
+  invisible-island.net/ncurses: ">=6.0"
+
+provides:
+  - bin/nano
+
+build:
+  dependencies:
+    freedesktop.org/pkg-config: "*"
+    gnu.org/automake: "*"
+    gnu.org/make: "*"
+    tea.xyz/gx/cc: "*"
+  working-directory: nano-{{ version.raw }}
```

Comment:
Or just `strip-components: 1` at the distributable level like all other packages do.